views/download_fluent_package.erb

```diff
@@ -0,0 +1,311 @@
+                  <a href="https://td-agent-package-browser.herokuapp.com/3/ubuntu/xenial/pool/contrib/t/td-agent">Xenial</a>,
+                  <a href="https://td-agent-package-browser.herokuapp.com/3/ubuntu/trusty/pool/contrib/t/td-agent">Trusty</a>
+                </li>
+                <li>Debian:
+                  <a href="https://td-agent-package-browser.herokuapp.com/3/debian/buster/pool/contrib/t/td-agent">Buster</a>
+                  <a href="https://td-agent-package-browser.herokuapp.com/3/debian/stretch/pool/contrib/t/td-agent">Stretch</a>,
+                  <a href="https://td-agent-package-browser.herokuapp.com/3/debian/jessie/pool/contrib/t/td-agent">Jessie</a>
+                </li>
+              </ul>
+              <a href="//docs.fluentd.org/v1.0/articles/install-by-deb">Installation Guide</a>
+            </td>
+          </tr>
+          <tr>
+            <td>
+              <center>
+                <img src="/images/download-icons/windows-logo.png"><br/>
+                Windows
+              </center>
+            </td>
+            <td>Windows Server 2012+ (64-bit)</td>
+            <td>
+              <span style="font-weight:bold">td-agent v<%=TD_AGENT_VERSIONS[:v4][:win]%></span>
+              <ul>
+                <li>
+                  <a href="https://td-agent-package-browser.herokuapp.com/4/windows">msi repository</a>
+                </li>
+              </ul>
+              <span style="font-weight:bold">td-agent v<%=TD_AGENT_VERSIONS[:v3][:win]%> (old stable)</span>
+              <ul>
+                <li>
+                  <a href="https://td-agent-package-browser.herokuapp.com/3/windows">msi repository</a>
+                </li>
+              </ul>
+              <a href="//docs.fluentd.org/v1.0/articles/install-by-msi">Installation Guide</a>
+            </td>
+          </tr>
+          <tr>
+            <td>
+              <center>
+                <img src="/images/download-icons/logo_mac.gif"><br/>
+                MacOS X
+              </center>
+            </td>
+            <td>OSX 10.11 or later</td>
+            <td>
+              <span style="font-weight:bold">td-agent v<%=TD_AGENT_VERSIONS[:v4][:mac]%></span>
+              <ul>
+                <li>
+                  <a href="https://td-agent-package-browser.herokuapp.com/4/macosx">dmg repository</a>
+                </li>
+              </ul>
+              <span style="font-weight:bold">td-agent v<%=TD_AGENT_VERSIONS[:v3][:mac]%> (old stable)</span>
+              <ul>
+                <li>
+                  <a href="https://td-agent-package-browser.herokuapp.com/3/macosx">dmg repository</a>
+                </li>
+              </ul>
+              <a href="//docs.fluentd.org/v1.0/articles/install-by-dmg">Installation Guide</a>
+              <br/>
+              <br/>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <br/>
+
+      <h3>Fluentd (v0.12, old stable version)</h3>
```

Comment:
This section should be separated to another page too, because it includes docker & k8s & gem which aren't td-agent.

Reply:
Removed docker and k8s entry.

(Strictly speaking, gem isn't distribution but kept it in)